Makefile

```diff
@@ -8,6 +8,7 @@ pull:
 
 build: pull
 	docker run -v ${pwd}:/root/cs3apis cs3org/cs3apis cs3apis-build -build-proto
+	chown -R `ls -ld . | awk '{print $$3 ":" $$4}'` docs
```

Comment:
Explanation please :)

Reply:
:-D sure - I'll add a comment in the file